cms/djangoapps/contentstore/views/preview.py

```diff
@@ -124,6 +125,28 @@ def user_id(self):
         return self._request.user.id
 
 
+class StudioPermissionsService(object):
```

Comment:
Thanks @cpennington. As we are up against a tight merge deadline, would just marking this as deprecated be sufficient to get a thumbs up? If you need a plan for replacing it then we'll have to leave this commit out of the PR so we can merge now.